instrumentation/net/http/otelhttp/handler.go

```diff
@@ -232,4 +233,4 @@ func WithRouteTag(route string, h http.Handler) http.Handler {
 		span.SetAttributes(semconv.HTTPRouteKey.String(route))
 		h.ServeHTTP(w, r)
 	})
-}
+}
```

Comment:
A new line character is needed at the end of the file.

Reply:
Added it to the file thanks